src/lib/es2020.intl.d.ts

```diff
@@ -295,30 +295,32 @@ declare namespace Intl {
         | "code"
         | "none";
 
-    type ResolvedDisplayNamesType =

```

Comment:
Will there be anybody using `ResolvedDisplayNamesType`? If so, it should stick around around as a simple alias of `DisplayNamesType`